other/bootstrap_daemon/docker/Dockerfile

```diff
@@ -11,42 +14,40 @@ RUN apt-get update && apt-get install -y \
       git \
       yasm \
       libsodium-dev \
+      python3" && \
+    export RUNTIME_PACKAGES="\
       libconfig-dev \
-      python3 \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
-
-# install toxcore and daemon
-WORKDIR /root/
-RUN git clone https://github.com/irungentoo/toxcore
-WORKDIR /root/toxcore/
-RUN ./autogen.sh
-RUN ./configure --enable-daemon
-RUN make -j`nproc`
-RUN make install -j`nproc`
-RUN ldconfig
-
-WORKDIR /root/toxcore/other/bootstrap_daemon/
-
-# add new user
-RUN useradd --home-dir /var/lib/tox-bootstrapd --create-home \
+      libsodium13" && \
+    apt-get update && apt-get install -y $BUILD_PACKAGES $RUNTIME_PACKAGES && \
+    git clone https://github.com/irungentoo/toxcore && \
+    cd toxcore && \
+    ./autogen.sh && \
+    ./configure --enable-daemon && \
+    make -j`nproc` && \
+    make install -j`nproc` && \
+    ldconfig && \
+    useradd --home-dir /var/lib/tox-bootstrapd --create-home \
       --system --shell /sbin/nologin \
       --comment "Account to run Tox's DHT bootstrap daemon" \
-      --user-group tox-bootstrapd
-RUN chmod 700 /var/lib/tox-bootstrapd
-
-RUN cp tox-bootstrapd.conf /etc/tox-bootstrapd.conf
-
-# remove all the example bootstrap nodes from the config file
-RUN N=-1 && \
+      --user-group tox-bootstrapd && \
+    chmod 700 /var/lib/tox-bootstrapd && \
+    cp other/bootstrap_daemon/tox-bootstrapd.conf /etc/tox-bootstrapd.conf && \
+    N=-1 && \
     while grep -q "bootstrap_nodes =" /etc/tox-bootstrapd.conf; \
     do \
       head -n $N tox-bootstrapd.conf > /etc/tox-bootstrapd.conf; \
       N=$((N-1)); \
-    done
-
-# add bootstrap nodes from https://nodes.tox.chat/
-RUN python3 docker/get-nodes.py >> /etc/tox-bootstrapd.conf
+    done && \
+    python3 other/bootstrap_daemon/docker/get-nodes.py >> /etc/tox-bootstrapd.conf && \
+    export AUTO_ADDED_PACKAGES="$(apt-mark showauto)" && \
+    apt-get remove --purge -y $BUILD_PACKAGES $AUTO_ADDED_PACKAGES && \
+    apt-get install -y $RUNTIME_PACKAGES && \
```

Comment:
Yes, that's correct. But libconfig-dev gets removed (from 'apt-mark show auto' and 'apt-get remove --purge') and I get
`/usr/local/bin/tox-bootstrapd: error while loading shared libraries: libconfig.so.9: cannot open shared object file: No such file or directory`
, and libsodium has been added for re-installation just to accompany libconfig.